common/src/main/scala/com/gu/media/aws/UploadAccess.scala

```diff
@@ -12,6 +12,10 @@ trait UploadAccess { this: Settings with AwsAccess =>
   lazy val uploadSTSClient = createUploadSTSClient()
 
   private def createUploadSTSClient() = {
+    if(!userUploadRole.startsWith("arn:")) {
```

Comment:
If you use the non-arn format you only get an "Unauthorized" error from Amazon which is quite confusing. After the second time it happened to me, I added a check to give a better exception 😄